### PR TITLE
Put Grafana dashboards in different folders

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
This PR changes Grafana settings to manage dashboards across multiple Grafana folders declaratively.

image

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/pipecd/values.yaml [hunk 1]
```diff
@@ -373,13 +373,35 @@ prometheus:
 # https://github.com/grafana/helm-charts/tree/main/charts/grafana
 grafana:
   adminPassword: admin
+  dashboardProviders:
+    dashboardproviders.yaml:
+      apiVersion: 1
+      providers:
+        - name: 'Cluster-Stats'
```
Comment (nghialv) [inline manifests/pipecd/values.yaml:380]: nit: One vote from me to get rid of the suffix, just use `Cluster`, `Control-Plane`, `Piped`.

Comment (nakabonne) [reply manifests/pipecd/values.yaml:380]: Alright. I'm gonna adopt it as I feel the same way.

**Comment by nghialv** [comment]
I also wonder about the convention of the dashboard name.
Where should we use a plural noun or where should not. We are having `Nodes`, `Pods` but why `Prometheus`, `Go`.
So IMO, to be simple we don't use plural nouns at all.

**Comment by nakabonne** [comment]
>So IMO, to be simple we don't use plural nouns at all.

Looks neat. I apply it as well

**Comment by nakabonne** [comment]
Fixed.

**Comment by nghialv** [comment]
Nice.
/lgtm

**Comment by khanhtc1202** [comment]
🚀 
/approve

**Pull request merged**